frontend/public/style/_overrides.scss

```diff
@@ -362,6 +362,10 @@ h6 {
   }
 }
 
+.pf-c-pagination .pf-c-options-menu__menu {
```

Comment:
I guess this is necessary for the same reason as given in the comment above (`due to setting $pf-global--enable-reset to false`)

Reply:
Yeah.  Mind duplicating the comment?